packages/snaps-rpc-methods/src/index.ts

```diff
@@ -6,3 +6,5 @@ export {
 export * from './restricted';
 export { SnapCaveatType } from '@metamask/snaps-utils';
 export { selectHooks } from './utils';
+export * from './endowments';
+export * from './permission-utils';
```

Comment:
I don't think we need to nest `permissions.ts` in a `permissions-utils` folder 🤔

Reply:
Where would you like it to be? I'm trying to avoid adding it to the root and start having too many files there.

Reply:
I don't think one additional file in the root is too bad. We can move this if it becomes a problem later, but I don't see it being a problem right now

Reply:
As you wish. Moved to the root.